_layouts/page.html

```diff
@@ -2,13 +2,28 @@
 layout: default
 ---
 
+
 <div>
   <article role="article">
     {% if page.title %}
     <header>
       <h1 class="entry-title">{{ page.title }}</h1>
     </header>
     {% endif %}
+
+    {% if page.files %}
+      {% for zip in page.files %}
+        <a class="tutorial-filelist__heading" name="download-{{zip.zipname}}">Files for {{zip.zipname}}</a>
```

Comment:
[The name attribute is obsolete in `a` tags](https://developer.mozilla.org/en/docs/Web/HTML/Element/a#attr-name) (not in form elements, though).
You can use `id="download-{{zip.zipname}}"` instead.

Also, I personally wouldn't use an `a` element here, but a heading, so that there is an indication that this is a different section from what is strictly the tutorial copy. So something like, either:

```html
<h2 class="tutorial-filelist__heading" id="download-{{zip.zipname}}">Files for {{zip.zipname}}</h2>
```

or maybe:

```html
<section>
  <h1 class="tutorial-filelist__heading" id="download-{{zip.zipname}}">Files for {{zip.zipname}}</h1>
</section>
```

Or even:

```html
<section>
  <h1>Download the tutorial files!</h1>

  <p class="tutorial-filelist__heading" id="download-{{zip.zipname}}">Files for {{zip.zipname}}</p>
  <!-- Files here... -->
  <p class="tutorial-filelist__heading" id="download-{{zip.zipname}}">Files for {{zip.zipname}}</p>
  <!-- Files here... -->
</section>
```